week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -36,11 +55,20 @@ Write a function that:
 - the numbers must be rounded to 2 decimal places
 - numbers greater 100 must be replaced with 100
 */
-
 function formatPercentage(arr) {
-  
+  let strArr = [];
+  let numRound = [];
```

Comment:
This works, and could be useful if we also want the number values as well as the strings, but we only actually use each element of this array during one iteration of the loop.

Instead of creating an array which we fill in (which may become very large, and use lots of memory, if the input array was large), we could instead use a single variable scoped to inside the loop, maybe called `numRound` which is a number type, rather than an array.